GUI/src/main/kotlin/ui/screens/DiffScreen.kt

```diff
@@ -60,26 +62,26 @@ fun DiffScreen(state: MutableState<AppState>) {
         }
 
         // #####   Titles   #####
-        Row(modifier = Modifier.fillMaxWidth().weight(0.1f)) {
+        Row(modifier = Modifier.fillMaxWidth().weight(0.08f)) {
             TextTitle(text = "Reference Video")
             TextTitle(text = "Diff")
             TextTitle(text = "Current Video")
         }
 
-        Row(modifier = Modifier.fillMaxWidth().weight(0.1f)) {
+        Row(modifier = Modifier.fillMaxWidth().weight(0.08f)) {
```

Comment:
This leads to the statistical information being cut off and to visible. But I think, this is something for a new ticket for next sprint, the text should be responsive anyway. As well as the text in the buttons etc. So I guess this is fine.